pkg/build/testdata/missing-mixins-token-expected-output.Dockerfile

```diff
@@ -1,12 +1,12 @@
 # syntax=docker/dockerfile-upstream:1.4.0
 FROM ubuntu:light
 ARG BUNDLE_DIR
-COPY mybin /cnab/app/
-ARG BUNDLE_DIR
 ARG BUNDLE_UID=65532
 ARG BUNDLE_USER=nonroot
 ARG BUNDLE_GID=0
 RUN useradd ${BUNDLE_USER} -m -u ${BUNDLE_UID} -g ${BUNDLE_GID} -o
+ARG BUNDLE_DIR
```

Comment:
Did these get moved due to a permission error, or there was no reason (okay if there isn't, just checking)

Reply:
> Did these get moved due to a permission error, or there was no reason (okay if there isn't, just checking)

Without this fix line 3-7 was inserted at the end of the file, causing the permission error described in the issue. I don't know if this answers you question, otherwise please let me know

Reply:
All good, thank you :)